packages/prosess-tilbakekreving-vedtak/src/types/VedtaksbrevAvsnitt.ts

```diff
@@ -1,5 +1,5 @@
 export type VedtaksbrevAvsnitt = {
-  avsnittstype: string;
+  avsnittstype: `${AvsnittType}`;
```

Comment:
Jøss. Denne syntaxen visste jeg ikke om.

Reply:
Å refaktorere enumene til string unions er kanskje ikke helt rett frem?

Reply:
Nei, jeg har holdt litt igjen på på gjøre alt til string unions for å begrense refaktureringen litt. I tilfelle med `AvsnittType` krever det å se litt på utformingen av FormValues der den brukes(formvalues er for øyeblikket `Record<string, Record<string, string> | string`.

Men jeg tenker at når vi får sterkere typing rundt forbi så kan vi skrive oss vekk fra enum der det er logisk!